### PR TITLE
[FSTORE-798][APPEND] Clean up beam pom.xml

### PR DESCRIPTION
**Pull request by davitbzh**

### CODE REVIEW

File: integrations/java/beam/README.md [hunk 1]
```diff
@@ -39,23 +42,24 @@ Currently, Beam support for Hopsworks feature store is experimental and only wri
...
-You can execute this notebook directly on Hopsworks cluster. Follow the documentation how to run [spark notebooks](https://docs.hopsworks.ai/3.1/user_guides/projects/jupyter/spark_notebook/)
-and [python notebooks](https://docs.hopsworks.ai/3.1/user_guides/projects/jupyter/python_notebook/).
+You can execute this notebook directly on Hopsworks cluster. Follow the documentation how to run [spark notebooks](https://docs.hopsworks.ai/3.3/user_guides/projects/jupyter/spark_notebook/)
+and [python notebooks](https://docs.hopsworks.ai/3.3/user_guides/projects/jupyter/python_notebook/).
 
 ## Data source
 Feature pipeline needs to connect to some data source to read the data to be processed. In this tutorial you will
-use publicly available topic `projects/pubsub-public-data/topics/taxirides-realtime`
+use publicly available topic `projects/pubsub-public-data/topics/taxirides-realtime`. Ensure that the associated 
+service account has the Pub/Sub Subscriber role.
```
Comment (kennethmhc) [inline integrations/java/beam/README.md:56]: I think it needs to be `Pub/Sub Admin`

Comment (davitbzh) [reply integrations/java/beam/README.md]: done

File: integrations/java/flink/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>hopsworks-tutorials</artifactId>
     <groupId>com.hopsworks.tutorials</groupId>
-    <version>3.3.0-SNAPSHOT</version>
+    <version>3.4.0-SNAPSHOT</version>
```
Comment (kennethmhc) [inline integrations/java/flink/pom.xml:8]: `<flink.version>1.17.0.0</flink.version>` this should be `1.17.0`

**Review by kennethmhc — CHANGES_REQUESTED** [review]

**Review by kennethmhc — APPROVED** [review]
LGTM!

**Review by kennethmhc — APPROVED** [review]

**Review by Maxxx-zh — APPROVED** [review]

**Pull request merged**